action.yml

```diff
@@ -172,10 +172,62 @@ inputs:
                  command length limit. The minimum value is 0 and the maximum value is 30000. A value of
                  0 means that every file will be signed with an individual call to signtool.
     required: false
+  cache-dependencies:
+    description: A boolean value (true/false) that indicates if the dependencies for this action should be
+                 cached by GitHub or not. When using self-hosted runners, caches from workflow runs are stored
+                 on GitHub-owned cloud storage. A customer-owned storage solution is only available with GitHub
+                 Enterprise Server. When enabled, this option can reduce the duration of the action by at least
+                 1 minute.
+    required: false
+    default: 'true'
 
 runs:
   using: 'composite'
   steps:
+    - name: Set $PSModulePath
+      id: set-psmodulepath
+      shell: 'pwsh'
+      run: |
+        $defaultPath = $env:PSModulePath -split ';' | Select-Object -First 1
+        "PSMODULEPATH=$defaultPath" | Out-File -FilePath $env:GITHUB_OUTPUT -Append
+      if: ${{ inputs.cache-dependencies == 'true' }}
+
+    - name: Cache TrustedSigning PowerShell module
+      id: cache-module
+      uses: actions/cache@v3
+      env:
+        cache-name: cache-module
+      with:
+        path: ${{ steps.set-psmodulepath.outputs.PSMODULEPATH }}\TrustedSigning\0.3.18
```

Comment:
Maybe `0.3.18` and other version should be put in a variable instead?

Reply:
Created a variable via build output in an earlier step.